patches/chromium/refactor_restore_base_adaptcallbackforrepeating.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Is there a tracking issue for the work to remove our usage of AdaptCallbackForRepeating?

Reply:
@MarshallOfSound this was discussed in the upgrades meeting, I'll be adding a tracking issue today

Reply:
Heh fair, sorry was climbing the hill again 😆 ⛰️

Reply:
https://github.com/electron/electron/issues/29739